content/rs/connections/_index.md

```diff
@@ -0,0 +1,8 @@
+---
+Title: Manage client connections
+linkTitle: Connections
+description:
+weight: 70
+categories: ["RS"]
+---
+Learn how to connect your application to a Redis database hosted by Redis Enterprise Software and test your connection.
```

Comment:
I would add links to the articles below this so it's easy for people to navigate without using the TOC (in case they are using smaller screens or screen readers).

Reply:
sounds good!

Reply:
Updated, @kaitlynmichael. 

[Preview](https://docs.redis.com/staging/jira-doc-1229b/rs/connections/)